src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -291,7 +292,10 @@ export class SearchMenu extends React.Component {
                 disabled={!isUserInputValid()}
                 data-e2e-id="sitewide-search-submit-button"
                 className="vads-u-margin-left--0p5 vads-u-margin-y--1 vads-u-margin-right--1 vads-u-flex--1"
-                onClick={() => handleSearchEvent()}
+                onMouseDown={() => {
+                  event.preventDefault();
+                  handleSearchEvent();
+                }}
```

Comment:
Seems a little odd that we would want to initiate a search on mouse down rather than on click, but I trust this is the desired UX...?

Reply:
Wondering about the switch to `onMouseDown` as well ^

Also I think that we need `event` to be defined as an arg to the `onMouseDown ` handler (currently I think it is pulling `event` out of the global scope, and IDK if we can always count on that.)

Reply:
So the reason for the switch to onMouseDown has to do with some of the joyful idiosyncrasies of Downshift + accessibility concerns + analytics tagging requirements. 

I am happy to go into explicit detail, but long story short the onMouseDown is to allow us to start a search and then prevent the onFocus event from clearing the suggestions so that they are present for the analytics tagging. I will definitely research if I need to add the event arg so as not to cause some issues with global scope

Reply:
based on feedback though I am going to look at some other options and circle back!

Reply:
also as a note above, there is still an event handler taking care of any keyboard handled submits on this form, this is just affecting mouse clicks of this button.
You can still fire off a search by pressing enter while focusing the button, pressing enter while focusing the input field, clicking on of the search suggestions, and now this, clicking the search button

Reply:
> I am happy to go into explicit detail, but long story short the onMouseDown is to allow us to start a search and then prevent the onFocus event from clearing the suggestions so that they are present for the analytics tagging. I will definitely research if I need to add the event arg so as not to cause some issues with global scope

Oof, I don't think I need more detail than that! Sounds a little messy. Thanks for the info!

>  I will definitely research if I need to add the event arg so as not to cause some issues with global scope

Yeah we were seeing `event` work there without defining it as an arg to the handler because of this, https://developer.mozilla.org/en-US/docs/Web/API/Window/event. I learned that awhile back when I also forgot to write `event` as an arg in my handler and was puzzled why I didn't get a runtime error.